README.md

```diff
@@ -1,7 +1,135 @@
+
+Getting Started
+--------------------------------------------------------------------------------
+
+In order to run this demo, you'll need a Google Cloud account, a MongoDB Atlas
+account, credentials for both services, and a working Terraform installation.
+
+A quick note on security: this demo uses your personal credentials for Google
+Cloud and a highly privileged API key for Atlas. It also configures Atlas
+to allow connections from any IP address. We do this to get you
+up and running as fast as possible, but in the real world you should follow
+Google Cloud's [best practices for service accounts](https://cloud.google.com/iam/docs/best-practices-service-accounts)
+and Atlas's [documentation](https://www.mongodb.com/docs/atlas/atlas-ui-authorization/)
+to properly secure your infrastructure.
+
+With that out of the way, let's get started!
+
+### MongoDB Atlas
+
+If you don't already have a Atlas account, [sign up here](https://www.mongodb.com/cloud/atlas/register).
+If you are prompted to create a database, look for the "I'll do this later" link
+in the lower left corner. Once you're logged in, click on "Access Manager" at
+the top and select "Organization Access".
+
+Select the "API Keys" tab and click the "Create API Key" button. Give your new
+key a short description and select the "Organization Owner" permission. Click
+"Next" and then make a note of your public and private keys. This is your last
+chance to see the private key, so be sure you've written it down somewhere safe.
+
+Next, click "Done" in the lower right, then click the gear icon in the upper left,
+between your organization name and the "Access Manager" link. This will take you
+to the Organization Settings screen. Copy your Organization ID and save it with
+your credentials.
+
+That's everything for Atlas. Now you're ready to move on to setting up Google Cloud!
+
+### Google Cloud
+
+If you don't already have a Google Cloud account, [sign up here](https://accounts.google.com/SignUp).
+We recommend using [Cloud Shell](https://cloud.google.com/shell) but if you prefer
+to work locally you can do that too. If you'd like to use Cloud Shell, just click
+this convenient button:
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://shell.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/terraform-mean-cloudrun-mongodb)
+
+This will open a Cloud Shell window and clone a copy of this repository. Cloud
+Shell has your credentials built in, and Terraform is already installed, so
+you're ready to move on to running the demo. You can also [open Cloud Shell the normal way](https://cloud.google.com/shell/docs/using-cloud-shell)
+but you'll need to clone this repo yourself.
+
+If you want to work locally, you'll need to [install the `gcloud` tool](https://cloud.google.com/sdk/docs/install).
+Confidential to Mac users: `gcloud` is also available [via homebrew](https://formulae.brew.sh/cask/google-cloud-sdk).
```

Comment:
Just saying `Mac users can also get gcloud via homebrew` and taking out the word confidential sounds more natural to me. Awesome that you included a link to the cask though, definitely keep that.

Reply:
Legit! Will edit.